bite-size-notes/week-8-e-Bayes Rule-RBE-RBF.md

```diff
@@ -0,0 +1,94 @@
+$$P(Y_1|Y_2,X)=P(Y_1|X)$$
+$$P(Y_2|Y_1,X)=P(Y_2|X)$$
+
+## Recursive Bayesian Filter
+
+Say we have $X$ which changes over time and assuming the system is *Markov*, meaning "the next state depends on only the current state", we can write this as:
+$$P(X_0,X_1,X_2,\cdots,X_K) = P(X_0)\prod_{i=1}^KP(X_i|X_{i-1})$$
+
+However, this is the Markov chain without control. When we insert control, expressed as $P(X_K|X_{K-1}, U_{K-1})$ we get a *Markov Decision Process*. If we include observations and assume some/all of $X$ cannot be observed, we have a *Partially Observable Markov Decision Process*.
+
+The core of the Recursive Bayesian Filter is a two-step process: predict and update.
+
+### Predict
+
+$$P(X_K,X_{K-1}|Y_{1:K-1}) =  P(X_K|X_{K-1},Y_{1:K-1})P(X_{K-1}|Y_{1:K-1})$$
+$$P(X_K|X_{K-1})P(X_{K-1}|Y_{1:K-1})$$
+$$P(X_K|Y_{1:K-1}) = \int_{X_{K-1}} P(X_K, X_{K-1}|Y_{1:K-1})dX_{K-1}$$
+
+### Update
+
+$$P(X_K|Y_{1:K}) = P(X_K|Y_K,Y_{1:K-1})$$
+$$BR \propto P(Y_K|X_K,Y_{1:K-1})P(X_K|Y_{1:K-1})$$
+$$\propto P(Y_K|X_K)P(X_K|Y_{1:K-1})$$
+
+where $P(X_K|Y_{1:K-1})$ is from the predict step.
+
+## Conclusion: Why this is important
+
+The Recursive Bayesian Filter is a fundamental tool for state estimation and tracking due to its ability to handle noise and uncertainty in measurements. It is particularly useful in real-time applications since it continuously updates the probability distribution with new data, making the estimation system robust and accurate. Finally, its importance is highlighted given that several other more complex tools build upon the Recursive Bayesian Filter:
+- **Kalman Filter** - A specific implementation of the RBF for linear Gaussian systems
+- **Extended Kalman Filter** - An extension of the Kalman filter for nonlinear systems
+- **Unscented Kalman Filter** - A modification of the EKF to address some of its limitations, specifically attempting to better capture the mean and covariance estimates of the state distribution
+- **Particle Filter** - A representation of the posterior distribution using a set of randomly chosen weighted samples or particles, important for non-Gaussian and nonlinear systems
+- **Multiple-Model Estimators** - A combination of models to describe different modes of the system dynamics
+- **Grid-Based Filters** - Filters that discretize the state space and apply the RBF within said discretized space
+- **BraMBLe (Bayesian Multiple-Blob Tracker)** - A tracker using particle filters to handle multiple targets in tracking scenarios
+- **Importance Sampling** - A technique used within Particle Filters to weight samples according to their likelihood, allowing for efficient estimation of the posterior distribution in high-dimensional spaces.
+
+## References
+[1] Leung, Karen. “Linear Multivariable Control” Lecture, University of Washington, Seattle, 05-13-2024, 05-16-2024.
+
+[2] [Rubinstein, Michael. "Introduction to Recursive Bayesian Filtering", MIT](https://people.csail.mit.edu/mrub/talks/filtering.pdf)
```

Comment:
These references never appear in the body of the text. Make sure to indicate which parts of the notes were taken from these references!

Reply:
I added the reference to the end since the presentation is where I got the further applications of the Recursive Bayesian Filter. Almost everything else comes from class notes.